NineChronicles.Headless/GraphTypes/ActionMutation.cs

```diff
@@ -86,8 +83,8 @@ public ActionMutation()
                             name = avatarName,
                         };
 
-                        var actions = new PolymorphicAction<ActionBase>[] { action };
-                        Transaction<PolymorphicAction<ActionBase>> tx = blockChain.MakeTransaction(privateKey, actions);
+                        var actions = new NineChroniclesActionType[] { action };
+                        Transaction<NineChroniclesActionType> tx = blockChain.MakeTransaction(privateKey, actions);
```

Comment:
Maybe, we should unify the alias name, `NineChroniclesActionType` or `NCAction`

Reply:
Applied. https://github.com/planetarium/NineChronicles.Headless/pull/481/commits/2538e1de46714dd554bc8d723787675e01287e3e